ops/pebble.py

```diff
@@ -685,3 +730,14 @@ def get_plan(self) -> Plan:
         """Get the Pebble plan (currently contains only combined services)."""
         result = self._request('GET', '/v1/plan', {'format': 'yaml'})
         return Plan(result['result'])
+
+    def get_services(self, names: List[str] = None) -> List[ServiceInfo]:
+        """Get the service status for the configured services.
+
+        If names is specified, only fetch the service status for the services
+        named.
+        """
+        if names is None:
+            names = []
+        result = self._request('GET', '/v1/services', {'names': ','.join(names)})
```

Comment:
I would have expected names not to be sent if empty.

Reply:
Good call - updated.